From e2248aca8630bd375a1a5f06fa51e4aba21d9e40 Mon Sep 17 00:00:00 2001
From: Eric Harney <eharney@redhat.com>
Date: Mon, 24 Jun 2013 14:15:38 -0400
Subject: [PATCH] Adjust pip-requires for EL6

Remove dep on wsgiref -- doesn't appear to be used by Cinder.
Allow sqlalchemy-migrate 0.6
Relax version requirements for babel, paramiko, lxml
---
 tools/pip-requires | 9 ++++-----
 1 file changed, 4 insertions(+), 5 deletions(-)

diff --git a/tools/pip-requires b/tools/pip-requires
index 2b816d1..acecdcd 100644
--- a/tools/pip-requires
+++ b/tools/pip-requires
@@ -1,15 +1,15 @@
 amqplib>=0.6.1
 anyjson>=0.2.4
 argparse
-babel>=0.9.6
+babel
 eventlet>=0.9.12
 greenlet>=0.3.1
 iso8601>=0.1.4
 kombu>=1.0.4
 lockfile>=0.8
-lxml>=2.3
+lxml
 oslo.config>=1.1.0
-paramiko>=1.8.0
+paramiko
 paste
 pastedeploy>=1.5.0
 python-glanceclient>=0.5.0,<2
@@ -18,8 +18,7 @@ python-swiftclient>=1.2,<2
 routes>=1.12.3
 six
 sqlalchemy>=0.7,<=0.7.99
-sqlalchemy-migrate>=0.7
+sqlalchemy-migrate
 stevedore>=0.7
 suds>=0.4
 webob>=1.0.8
-wsgiref>=0.1.2
